src/components/modal/Modal.js

```diff
@@ -31,7 +31,7 @@ class Modal extends React.Component {
       <div
         className={classNames(
```

Comment:
Since there is no classes depending on variables, I would remove too the function `classNames()`

Reply:
@tureey one newbie general question. As this is not related to this issue, I should change in this issue? 
Talking in general cases, I added a class to the modal during this issue, but I haven't think/designed how the modal has to be or I don't know if there's another issue related to this problem. I should change here? how it's usually done, I should fix? I should ask before doing it? Of course, I have no problem in doing that! 
Thank you!

Reply:
Oh sure! :+1: 

IMHO if changing something that is not related to a task takes some time... We should not do it. There is not point of.. "blocking" a `PR` because of another thing that is not related to what we were doing.

On the other hand. I'm the kind of guy who fixes things every time I open a file (clean up the house):
- Format
- Variable names
- Quick wins
- etc...

In this case, the only thing required is to remove the `classNames()` function and its `import`. Personally I would do it in this `PR` because it will take @crysfel between 4h-12h to approve the `PR`. 

If you can't or even if you want, it's OK since it was not the main issue for this `PR` I'll do it tomorrow or we can open an issue :smile:

Reply:
Thank you @tureey for your very clear explanation. I understand how to proceed from now on.
If I'm able to detect those mistakes has to be seen! 😆 
I've commited the changes!
Thank you!

Reply:
It was not a mistake! :smirk:  well done again!